src/util/gov.ts

```diff
@@ -53,112 +106,206 @@ export const decodeContent = (content?: Any): PropDecoded => {
   if (!content) {
     return emptyProposal;
   }
-  switch (content.typeUrl) {
+  let url = content.typeUrl
+  let value = content.value
+  if (url == "/cosmos.gov.v1.MsgExecLegacyContent") {
```

Comment:
Instead of hardcoding path, think you can `import { TxTypes } from "./tx"` and use TxTypes.MsgGovExecLegacyContent instead.

Reply:
resolved in https://github.com/Switcheo/carbon-js-sdk/pull/440/commits/1d2943930756fb41647341556612ec627fc465c6